src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Configuration/IProjectConfigurationDimensionsProvider5.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Microsoft.VisualStudio.ProjectSystem
 {
-    internal interface IProjectConfigurationDimensionsProvider4 : IProjectConfigurationDimensionsProvider3
+    internal interface IProjectConfigurationDimensionsProvider5 : IProjectConfigurationDimensionsProvider3
```

Comment:
Why the rename?

Reply:
Because it clashed with the existing one from CPS.